storybooknative/src/styled/__tests__/index.test.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+/* eslint-disable no-return-assign */
+import { Platform } from "react-native";
+
+import { createPlatformStyles } from "../index";
+
+let originalPlatform;
+// $FlowFixMe
```

Comment:
```suggestion
beforeEach(() => {
  originalPlatform = Platform.OS;
});
afterEach(() => {
  Platform.OS = originalPlatform;
});
```
and that means top `eslint-disable` is not necessary anymore.

Reply:
done